sample/src/main/res/layout/activity_main.xml

```diff
@@ -9,6 +9,25 @@
       android:layout_width="wrap_content"
       android:layout_height="wrap_content"
       android:text="@string/hello_world"/>
+  <TextView
+      android:id="@+id/repeated_view_1"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:visibility="gone"
```

Comment:
Wep @Sloy how's the approach to this suggestion? It's something that we can do? Or it's too hard for this PR?